independent-publisher-connectors/GitLab/apiDefinition.swagger.json

```diff
@@ -0,0 +1,703 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v4",
+    "title": "GitLab",
+    "description": "A connector for the GitLab REST API.",
```

Comment:
@RoyPaarMS Please provide a detailed description for the connector which can be understandable by everyone, and it should describe concisely the main purpose and value offered by your connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
Corrected